Resources/Maps/Shuttles/infiltrator.yml

```diff
@@ -3726,19 +3726,19 @@ entities:
     - type: Transform
       pos: 5.6633797,-17.565443
       parent: 1
-- proto: NuclearBombUnanchored
+- proto: NuclearBombSyndicate
   entities:
-  - uid: 509
+  - uid: 510
     components:
     - type: Transform
       pos: -2.5,-12.5
       parent: 1
-- proto: NukeCodePaper
+- proto: NukeCodePaperStation

```

Comment:
?

Reply:
This is done so that the nuclear bomb in Shuttle Syndicate is always provided with its own codes.